taniumrest_connector.py

```diff
@@ -737,11 +735,11 @@ def _question_result(self, timeout_seconds, results_percentage, endpoint, action
             # reformat response data to simplify data path
             if data.get("result_sets", [])[0].get("columns"):
                 rows = data.get("result_sets")[0].get("rows")
-                for i in range(len(rows)):
```

Comment:
The variable 'i' was conflicting with the outer for loop, hence updating it with 'j'.